pkg/rpc/server/server.go

```diff
@@ -1153,7 +1145,7 @@ chloop:
 			resp.Payload[0] = b
 		case execution := <-s.executionCh:
 			resp.Event = response.ExecutionEventID
-			resp.Payload[0] = result.NewApplicationLog(execution, util.Uint160{})
+			resp.Payload[0] = result.NewApplicationLog(execution)
```

Comment:
`transaction_executed` event in `wsclient_test.go` should also be updated.

Reply:
Fixed.